CONTRIBUTING.md

```diff
@@ -75,3 +75,28 @@ npm run install:packages --force-local
 
 ## The package publish, deprecation, and alpha publication process
 <!-- TODO: Add this, also consider where to mention npm linking, either here or in the packages README -->
+### Alpha Components
+
+#### Description
+
+The purpose of an Alpha Component is to allow an engineer to work on a component, merge it into Psammead and publish it to npm before it is ready for use in production.
+
+#### Use Cases
+
+The accepted uses of an Alpha Component are:
+
+- When engineering do not have all of the acceptance or accessibility criteria yet but want to make a start on building the component (with the agreement they may need to change it based on the finalised criteria).
+- When a component needs to be merged into Psammead and published, then pulled into Simorgh in order to UX review or a11y test it properly.
+- This means that an Alpha component does not need a UX review or a11y swarm when it is first merged into Psammead, only when it needs to come out of Alpha.
+
+#### Workflow
+
+- A new component is created in Psammead, the version is set to `0.1.0-alpha.1`, and "tag": "alpha" is added to the publish config in the package.json, e.g [alpha tag example](https://github.com/bbc/psammead/commit/64d7aa18a6b6d0861c9fdcd1b88047e634376bb7#diff-19d4d1f1939749aaffae0b8080cc09f4R37-R38)
+- An alpha component warning is added to the top of the README, e.g [alpha warning example](https://github.com/bbc/psammead/commit/eb794cb4b18eb3c0f5857bc5e5476f92fbc7c1cc#diff-4124748292798b73a7d123a33fc255faR1-R3)
+- The component can then be worked on, merged into latest in Psammead and published to npm under the alpha tag.
```

Comment:
Can the first three points (and possibly six and nine), which are most workflowy, be ordered and separate from the less-workflowy points below? If they can, I feel like this would make it super simple to follow.

Reply:
Agree with @simonsinclair , perhaps a section like "Prerequisites" or something. I think six and nine can be included in the workflow section, the other points seem more like warnings or just provides more context as opposed to being a step in the workflow.